src/modules/ModuleFarming/Pool.vue

```diff
@@ -196,7 +199,16 @@ function openRoiCalculator() {
           class="space-y-4"
         >
           <div
-            v-if="!enabled"
+            v-if="!dexStore.isWalletConnected"
+            class="flex items-center space-x-6"
+          >
+            <TheWalletConnect
+              v-if="!dexStore.isWalletConnected"
+              size="md"
+            />
```

Comment:
`<TheWalletConnect>` is intended to be used only in a single place. `The` in its name stands exactly for it.

If we need to duplicate this button, the solution to re-use this component as-is isn't ok. The logic of connection should be run in a single place without duplication.

To solve this, we should use a single component which does it, and external buttons around the app that somehow interact with this component - via Pinia or other custom storage API (custom provide/inject mechanism, for example).